smartdashboard/utils/ManifestReader.py

```diff
@@ -190,3 +191,24 @@ def create_filereader(path: str) -> ManifestFileReader:
             title="Manifest file could not be decoded.", file=path, exception=jde
         ) from jde
     return manifest_file_reader
+
+
+def get_manifest_path(
+    directory: t.Optional[pathlib.Path], default: pathlib.Path
+) -> str:
+    """Get the manifest path using the directory
+    path passed in from the command line arguments.
+
+    :param directory: An experiment directory
+    :type directory: t.Optional[pathlib.Path]
+    :param default: Default path for testing
+    :type default: pathlib.Path
+    :return: Manifest path
+    :rtype: str
+    """
+
+    if directory is not None:
+        manifest_path = directory / ".smartsim/telemetry/manifest.json"
+    else:
+        manifest_path = default
```

Comment:
I'm going to return an empty string in this else case because I still want the dashboard to have the opportunity to raise a `Manifest Does Not Exist` error if a user doesn't provide a directory argument to parse.

Reply:
Just kidding, it's an empty `pathlib.Path`!